model_analyzer/config/generate/coordinate.py

```diff
@@ -74,6 +74,12 @@ def __eq__(self, other: Any) -> bool:
                 return False
         return True
 
+    def __lt__(self, other: Any) -> bool:
+        for i, v in enumerate(self._values):
+            if v != other[i]:
```

Comment:
I don't understand what the _lt__ is trying to evaluate? I would have thought it would be that each entry in other is less than it's counterpart in self.  So, why do you need to check that the values are not equal first? 

Should this just be:
```
for i, v in enumerate(self._values):
    if v > other[i]
        return False
return True

Reply:
I just need a way to compare coordinates so they can be sorted in a deterministic way. My method treats them like a number. 0,9,9,9 is less than 1,0,0,0 (999 is less than 1000). Your proposal is not deterministic. [1,0] compared with [0,1] would return false regardless of the order I put them in.

That all being said, this shouldn't be added to the real class just to be used in a test. Let me try to move the logic into a test file.

Reply:
this code is gone (and simpler code is now in the test file)